samples/apps/bookinfo/src/mysql/mysqldb-init.sql

```diff
@@ -3,8 +3,6 @@
  * mysql -h 127.0.0.1 -ppassword < mysqldb-init.sql
  */
 USE test;
-CREATE TABLE price (name VARCHAR(20), price INT);
```

Comment:
we could keep this for later even if it's not used yet

Reply:
:(. I have already pushed the docker image..

Reply:
no problem. it's fine